baby-bear/src/baby_bear.rs

```diff
@@ -421,6 +422,18 @@ fn monty_reduce(x: u64) -> u32 {
     x_sub_u_hi.wrapping_add(corr)
 }
 
+/// Create a BabyBear element from something already in Monty Form.
```

Comment:
I think lines 425 - 436 might not be needed anymore and if not we should delete them. (Don't want to let people access the internal representation of a BabyBear Element.